### ISSUE TITLE
Mention additional requirements

### ISSUE THREAD

**Issue by mirh** [GPUOpen-LibrariesAndSDKs/TAN#1]
Shouldn't you mention you need OpenCL (I guess your APP SDK runtime libraries)?

EDIT: and Microsoft Foundation Classes for at least one of the samples

**Comment by GeoffPark** [comment]
To compile, you do need OpenCL headers. The OpenCL runtime is included with Radeon drivers. Note that TAN library also has a fully functional, though slower, CPU mode which does not require OpenCL runtime or a GPU.

**Comment by mirh** [comment]
> To compile, you do need OpenCL headers.

Yes, I was pointing out that this requirement is not mentioned in your readme prerequisites.

**Comment by GeoffPark** [comment]
Thanks for pointing that out. We will correct the readme in next update.

-Geoff
